unified-runtime/source/adapters/hip/command_buffer.hpp

```diff
@@ -128,6 +128,8 @@ struct ur_exp_command_buffer_handle_t_ : ur::hip::handle_base {
   // Atomic variable counting the number of reference to this command_buffer
   // using std::atomic prevents data race when incrementing/decrementing.
   std::atomic_uint32_t RefCount;
+  // Track the event of the current graph execution.
```

Comment:
Can we expand the comment on why we need this, i.e as a workaround for HIP limitation